frontend/packages/patternfly/src/components/notification-drawer/notification-drawer.tsx

```diff
@@ -0,0 +1,93 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+  EmptyStateVariant,
+  Title,
+} from '@patternfly/react-core';
+import {
+  Drawer,
+  DrawerContent,
+  DrawerPanelContent,
+} from '@patternfly/react-core/dist/js/experimental/components/Drawer';
+
+import NotificationDrawerHeading from './notification-drawer-heading';
+import NotificationTypeHeading from './notification-type-heading';
+
+export enum NotificationTypes {
+  info = 'info',
+  warning = 'warning',
+  critical = 'danger',
+  success = 'success',
+  update = 'update',
+}
+
+const emptyState = (toggleExpanded) => (
+  <EmptyState variant={EmptyStateVariant.full} className="co-status-card__alerts-msg">
+    <Title headingLevel="h5" size="lg">
+      No critical alerts
+    </Title>
+    <EmptyStateBody>
+      There are currently no critical alerts firing. There may be firing alerts of other severities
+      or silenced critical alerts however.
+    </EmptyStateBody>
+    <EmptyStateSecondaryActions>
+      <Link to="/monitoring/alerts" onClick={toggleExpanded}>
+        View all alerts
+      </Link>
+    </EmptyStateSecondaryActions>
+  </EmptyState>
+);
+
+const NotificationDrawer: React.FC<NotificationDrawerProps> = ({
+  isExpanded,
+  children,
+  toggleExpanded,
+  alertData = [],
+  updateData = [],
+}) => {
+  const [isAlertExpanded, toggleAlertExpanded] = React.useState<boolean>(true);
+  const [isAvailableUpdateExpanded, toggleAvailableUpdateExpanded] = React.useState<boolean>(false);
+  const criticalAlerts = _.isEmpty(alertData) ? emptyState(toggleExpanded) : alertData;
```

Comment:
So `Critical Alerts` appears to be both `Critical` and `Warning` lvl.  Is this correct?  When I open the drawer in my cluster, I have `18` total Alerts.  I have to scroll to find the `Critical` ones.  It does seem like noise pushes the urgent issues blow the fold.  For example, I have a lot of `UsingDeprecatedAPIAppsV1beta2` warnings, prob don't care a ton about these, but the `KubePodCrashLooping` I probably do care about:

![Screen Shot 2020-01-19 at 10 32 50 PM](https://user-images.githubusercontent.com/280512/72696835-b07ecb00-3b0b-11ea-9894-ddf83b4c942c.png)

Reply:
@benjaminapetersen good point. @cshinn, do we want to display all alerts or just the warning/critical? Currently they are sorted by time...is that correct?